packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -214,3 +205,30 @@ export const useProductFullDetail = props => {
         quantity
     };
 };
+
+const getConfigPrice = (product, optionCodes, optionSelections) => {
```

Comment:
For consistency please put this function up next to the other helper functions in this file.